### ISSUE TITLE
Refine error on line audio cue

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#175348]
Screen reader users navigate through an editor via the arrow keys/mouse and are informed of errors with an audio cue when:

1. the cursor lands on a line with an error via line change
3. the cursor lands in an error range via a column change

When a screen reader user goes to a new line with an error, they hear the audio cue, but aren't advised as to whether or not the column they've landed on contains an error. 

In order to know this, they must explore the characters on that line. 

While working on the fix for https://github.com/microsoft/vscode/issues/174857, @kieferrm suggested it could be advantageous to have separate audio cues for each context. That would fix this case. 

Here's what I propose:

Separate audio cues for:
1. the cursor lands within an error range via line and/or column change
2. the cursor lands on a line with an error outside of the error range via line change

any thoughts are appreciated: @isidorn and @jooyoungseo

**Comment by meganrogge** [comment]
Could be something like: `errorOnLine` and `errorAtPosition`

**Comment by isidorn** [comment]
Sounds like a good idea to me. That way it would be easier to find the exact error location.
As long as we do not change the `errorOnLine` sound I think it is fine to introduce a new sound for `errorAtPosition`. It should be similar to the `errorOnLine` sound but slightly different. Ideally we would not play both.

**Comment by jooyoungseo** [comment]
As long as #175341 is resolved, I (PERSONALLY) don't think there is a significant need for this feature. In any event this feature is added, users may want to turn on either of them as @isidorn suggested.

**Comment by Michaelanna1** [comment]
`my `

**Comment by meganrogge** [comment]
I will leave this open to see if it gets any more requests

**Comment by meganrogge** [comment]
This could apply more broadly to all of the audio cues which impact a line.

**Comment by meganrogge** [comment]
Feedback "The audio sounds in two areas:
Moving when in a collapsed lie of code (video attached on this)
Similar issue when arrowing through an error.
In both cases when I am either on an error or in a collapsed content with a comment every SINGLE key press makes a sound.  This is particularly annoying when I am trying to fix an error and I am simply arrowing to what I know needs to be fixed… it as if every key press is saying ERROR, ERROR, ERROR when yes… I know this. 😊

What I would like in these cases is to have it work much more like NVDA does with spelling and grammar errors in word.  It is nice and tells you when enter and exit them.  And, in the case where you fix the issue and the error state goes away it would confirm this for me."
